examples/website-demos/virtual-event/src/components/Chat/ThreadHeaderUI.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useChannelActionContext, useChannelStateContext } from 'stream-chat-react';
+import { CloseX, Ellipse } from '../../assets';
+
+export const ThreadHeaderUI = () => {
+  const { thread } = useChannelStateContext();
+  const { closeThread } = useChannelActionContext();
+
+  return (
+    <div className='dm-header-container'>
+      <div className='dm-header-close' onClick={closeThread}>
+        <CloseX />
+      </div>
+      <div className='dm-header-title'>
+        <div>Thread Reply</div>
+        <div className='dm-header-title-sub-title'>
+          with {thread?.user?.name || thread?.user?.id || ''}
```

Comment:
early return before the jsx if no thread, so don't have to optionally chain here